vortex-mask/src/lib.rs

```diff
@@ -31,6 +32,7 @@ struct Inner {
     // Pre-computed values.
     len: usize,
     true_count: usize,
+    // i.e., the fraction of values that are true
```

Comment:
Can you rename this truthiness- it's less ambiguous

Reply:
Or density

Reply:
So you're saying a "more selective" mask has more true values?

That sounds wrong..

A highly selective mask should have fewer true values.

See why it's ambiguous?

Reply:
I thought of it as the fraction that is selected... but I understand your point. sigh.

`fraction_true`?